qiskit_nature/second_q/problems/__init__.py

```diff
@@ -25,24 +25,40 @@
    VibrationalStructureProblem
    LatticeModelProblem
 
+   PropertiesContainer
```

Comment:
Ideally I would like to modify the documentation of this class to also include `:inherited-members` (rather than the default as to not do this), because as of right now, the docs will not show all of the methods inherited from the `MutableSet` base class:
![screenshot_1659550086](https://user-images.githubusercontent.com/21973473/182678780-5b5fae0e-b8cb-41b6-ad5c-6cf61359a132.png)

Any suggestions on how to achieve this easily are highly welcome :slightly_smiling_face:

Reply:
> Ideally I would like to modify the documentation of this class to also include `:inherited-members` (rather than the default as to not do this), because as of right now, the docs will not show all of the methods inherited from the `MutableSet` base class: ![screenshot_1659550086](https://user-images.githubusercontent.com/21973473/182678780-5b5fae0e-b8cb-41b6-ad5c-6cf61359a132.png)
> 
> Any suggestions on how to achieve this easily are highly welcome 🙂

I made the `Bases: MutableSet` clickable going directly to  the official python documentation. This way this class doc. behaves like the others, without a need for an exception and one has complete information about `MutableSet` .